backend/x86_binary_emitter.ml

```diff
@@ -1682,7 +1682,14 @@ let assemble_line b loc ins =
         for _ = 1 to n do
           buf_int8 b 0
         done
-    | Hidden _ | Weak _ | Reloc _ | NewLine | Sleb128 _ | Uleb128 _ | Direct_assignment _ -> ()
+    | Hidden _ | Weak _ | NewLine | Sleb128 _ | Uleb128 _ | Direct_assignment _ -> ()
```

Comment:
This case should produce an error on `Sleb128`, `Uleb128` and `Direct_assignment` I would expect?

Reply:
fixed.